src/item_search.cpp

```diff
@@ -0,0 +1,64 @@
+#include "item.h"
+#include "material.h"
+#include "item_search.h"
+#include "cata_utility.h"
+
+std::pair<std::string, std::string> get_both( const std::string &a );
+std::function<bool( const item & )>
+item_filter_from_string(  std::string filter )
+{
+    if( filter.empty() ){
+        return [](const item&i){
+            return i.active || !i.active;
```

Comment:
It requires a true value, and the compiler flags don't accept not using the parameter. :P